src/tools/tool.select.js

```diff
@@ -64,12 +68,41 @@ module.exports = function(paper) {
       segments: true,
       stroke: true,
       fill: true,
+      //class: Path,
       tolerance: 5
     };
 
     hitOptions = _.extend(hitOptions, options);
     var hitResult = project.hitTest(event.point, hitOptions);
 
+    // // Debugging
```

Comment:
Some goes for this, get this removed before we ship, or add a debug flag to trigger it.